tls/s2n_client_hello.c

```diff
@@ -299,11 +326,35 @@ int s2n_process_client_hello(struct s2n_connection *conn)
     return S2N_SUCCESS;
 }
 
-int s2n_client_hello_recv(struct s2n_connection *conn)
+static int s2n_client_hello_process_cb_response(struct s2n_connection *conn, int rc)
```

Comment:
While in here: can you change to `S2N_RESULT` instead of posix `int` in function return?

Reply:
i updated the new static method. do you want me to update `s2n_client_hello_recv` as well?